extensions/__init__.py

```diff
@@ -0,0 +1 @@
+# This page intentionally left blank.
```

Comment:
Situation was actually opposite here. I added this file because otherwise mypy was complaining. But now I see that I misunderstood the idea. I have removed this file.